scipy/stats/tests/test_morestats.py

```diff
@@ -1687,35 +1672,44 @@ def test_auto(self):
         x = np.arange(0, 50) + 0.5
         y = np.arange(50, 0, -1)
         assert_equal(stats.wilcoxon(x, y),
-                     stats.wilcoxon(x, y, mode="exact"))
+                     stats.wilcoxon(x, y, method="exact"))
 
         # n <= 50: if there are zeros in d = x-y, use PermutationMethod
         pm = stats.PermutationMethod()
-        d = np.arange(0, 5)
+        d = np.arange(-2, 5)
```

Comment:
`method='exact'` gives the same result in this case. Let's test a case that allows us to distinguish between the two methods.